birdhouse/scripts/logging.include.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+BIRDHOUSE_COLOR=${BIRDHOUSE_COLOR:-1}
+if [ "${BIRDHOUSE_COLOR}" -eq "1" ]; then
+    BLUE=$(tput setaf 12)
+    GRAY=$(tput setaf 8)
+    CYAN=$(tput setaf 6)
+    PURPLE=$(tput setaf 5)
+    YELLOW=$(tput setaf 3)
+    GREEN=$(tput setaf 2)
+    RED=$(tput setaf 1)
+    NORMAL=$(tput sgr0)
+fi
+
+export MSG_DEBUG="${GRAY}DEBUG${NORMAL}:    "
```

Comment:
Here's an idea since we are centralizing the logging, let me know if it's too much.

When I browser the logs, I often need to know the time it happens.  How about creating new functions `log_debug()`, `log_info()` and so on, using those predefined color and prefix with a timestamp in front?

Reply:
Yes, that can be done. Will look at it around the end of the week.

Reply:
Done. I went for an even more generic `log {LEVEL} {message}` which can be controlled with `BIRDHOUSE_LOG_LEVEL` to limit the logging details as desired.